### PR TITLE
Extra task 1

### PR DESCRIPTION
**Pull request by alexiv-tn65**
dl extra_task_1

### CODE REVIEW

**Review by pacifikus — CHANGES_REQUESTED** [review]
Добрый день!

Спасибо за работу! В целом все хорошо, оставлю несколько комментариев о том, что можно улучшить:

- Логи от установки библиотек лучше удалять из ноутбука перед коммитом, так как они затрудняют восприятие, не давая полезной информации. 
- В jupyter notebook (равно как и в colab) для вывода датафрейма в выходную ячейку лучше использовать не `print`, а `display` - получается более эстетично и нативно
- При переходе в `evaluate` вы переводите модель в eval-режим, но потом не переводите обратно в train - в данном случае это критично, так как у вас в архитектуре есть `Dropout` и `BatchNorm1d` слои с разным поведением в eval и train режимах. Получается, после первой валидации у вас перестает работать dropout и batchnorm...
- Если исправить пункт выше, то модель как будто начинает страшно переобучаться... Думаю, стоит внести исправления и проверить/подебажить архитектуру

**Comment by alexiv-tn65** [comment]
Добрый день!
Спасибо за комментарий @pacifikus . Исправил как мне кажется все ошибки, в т.ч. добавил перевод модели обратно в train режим.  Внес изменения в архитектуру: изменил функцию активации на GeLU, удалил Batch Normalization, добавил еще Dropout с вероятностью 0.4 после скрытого слоя, изменил соотношение hidden_size входного и остальных слоев, уменьшил hidden_size входного и соответственно всех остальных слоев. Т.к. на мой взгляд был избыток нейронов для решения  задачи и  возможно сетка по этой причине переобучалась.

**Review by pacifikus — APPROVED** [review]
Добрый день!

Спасибо за работу! Все отлично!
